lib/src/usage_impl_io.dart

```diff
@@ -130,17 +130,17 @@ class IOPersistentProperties extends PersistentProperties {
   }
 
   @override
-  dynamic operator [](String key) => _map[key];
+  dynamic operator [](String key) => _map![key];
 
   @override
   void operator []=(String key, dynamic value) {
-    if (value == null && !_map.containsKey(key)) return;
-    if (_map[key] == value) return;
+    if (value == null && !_map!.containsKey(key)) return;
```

Comment:
I'd be inclined to mark `_map` late so you can drop all of these bangs.

Reply:
done